### ISSUE TITLE
Scalafmt does not indent type members when there are more than one

### ISSUE THREAD

**Issue by pjrt** [scalameta/scalafmt#862]
- **Version**: 0.6.6
- **Configuration**:
```
style = default
```

## Problem

Scalafmt formats code like this:
```scala
final case class SomeCaseClass[N <: Node,
R <: SomeLongUpperBound: Ordering: Parser](projectOrdering: N => R)
```

## Expectation

I would like the formatted output to look like this:
```scala
final case class SomeCaseClass[N <: Node,
  R <: SomeLongUpperBound: Ordering: Parser](projectOrdering: N => R)
```

Or something similar to that.

## Notes

1. This issue happens for `def`s too, not just classes. 
2. Could/Should `verticalMultiline` be extended to type members?

**Comment by pjrt** [comment]
Addition: It seems to only happen when there are more than one type member. 

```
final case class SomeCaseClass[X, SomeVeryVeryLongLONGlongTypeTypeOrderingParser](projectOrdering: N => R)
```

yields

```
final case class SomeCaseClass[X,
SomeVeryVeryLongLONGlongTypeTypeOrderingParser](projectOrdering: N => R)
```

We currently have a test for type members in classes, but it tests a single type member.

**Comment by pjrt** [comment]
This shouldn't have been closed. #864 did NOT fix this for non-vertical-multilined classes. So this issue still exists. 

I wanted to solve the problem there first because

1. I'm using it :P 
2. I wanted to split the way we do this for non-vertical-multilines to make it easier, and since I have little knowledge of how it currently works I thought getting my feet wet with what I do know was the better strategy. 

Opening again.

**Comment by pjrt** [comment]
This one is still happening. But I won't be fixing it since I use VerticalMultiline and it is fixed there. @olafurpg

**Comment by olafurpg** [comment]
This is a bug that should be fixed. I took a quick look but I couldn't figure out why it's happening

**Comment by poslegm** [comment]
@pjrt @olafurpg looks like fixed in 2.0.0-RC8:

```

~ scalafmt --stdin --config-str "style = default"
final case class SomeCaseClass[X, SomeVeryVeryLongLONGlongTypeTypeOrderingParser](projectOrdering: N => R)
final case class SomeCaseClass[
    X, SomeVeryVeryLongLONGlongTypeTypeOrderingParser
](projectOrdering: N => R)
```

**Comment by pjrt** [comment]
Nice! Closing then.
